sql/moz-fx-data-shared-prod/accounts_db_external/fxa_recovery_phones_v1/schema.yaml

```diff
@@ -0,0 +1,17 @@
+fields:
+- name: uid
+  type: INTEGER
```

Comment:
I didn't catch that the `uid` columns are actually strings, not integers. Fixing this in #7126.